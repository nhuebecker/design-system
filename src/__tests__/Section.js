import React from 'react'
import renderer from 'react-test-renderer'
import { Section } from '..'

describe('Section', () => {
  test('renders', () => {
    const json = renderer.create(<Section />).toJSON()
    expect(json).toMatchSnapshot()
  })
})
