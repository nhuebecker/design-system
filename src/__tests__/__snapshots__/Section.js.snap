// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section renders 1`] = `
.c2 {
  padding-left: 18px;
  padding-right: 18px;
  padding-top: 24px;
  padding-bottom: 24px;
  color: #ffffff;
  text-align: center;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c0 {
  text-align: center;
}

@media screen and (min-width:32em) {
  .c2 {
    padding-top: 36px;
    padding-bottom: 36px;
  }
}

<div
  className="c0 c1 c2"
/>
`;
